src/config.cpp

```diff
@@ -7,16 +7,24 @@
 #include "file_utils.h"
 #include "string_utils.h"
 #include "hud_elements.h"
+
 std::string program_name;
+std::vector<std::string> used_configs;
+static const char *mangohud_dir = "/MangoHud/";
+static std::string env_config = get_config_dir();
```

Comment:
That is a bit dangerous.

I don't think we should be using static non-trivial initializers.

Reply:
Does it look fine now?